src/app_service/service/chat/dto/chat.nim

```diff
@@ -44,9 +37,8 @@ type
 
 type ChatMember* = object
   id*: string
-  admin*: bool
   joined*: bool
-  roles*: seq[int]
```

Comment:
This is also a sign of that it should be a set of roles (not just one), same what @jrainville mentioned in https://github.com/status-im/status-desktop/pull/10909#discussion_r1214698036